### PR TITLE
Add Fastlane to automate mobile app releases: init + Beta builds

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

In order to make it easier to do more frequent app releases, we're going to use Fastlane to automate the process 🚀 

This PR is the first setup and only has support for building and sending beta builds to app stores. More is to come later :)

### Preview

Screenshot 2023-07-17 at 22 17 19

### CODE REVIEW

**Comment by tonypls** [comment]
> > ### ✅ Deploy Preview for _phenomenal-syrniki-c5ceaa_ ready!
> > _To edit notification comments on pull requests, go to your [Netlify site configuration](https://app.netlify.com/sites/phenomenal-syrniki-c5ceaa/configuration/deploys#deploy-notifications)._
> 
> @tonypls, should we delete the Netlify project now that you added the CloudFlare previews? Also no clue why it suddenly reappeared on this PR 😅

It's back from the dead! Yes I'll delete it now :)

**Comment by madsnedergaard** [comment]
@tonypls, wanna review this one so we can make a new release together this week? :)

File: mobileapp/Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
```
Comment (tonypls) [inline mobileapp/Gemfile:1]: We run Ruby now 🎉 💎

Comment (madsnedergaard) [reply mobileapp/Gemfile:1]: Yup, more languages = more fun ❤️

**Pull request merged**